docs/source/basicusage.rst

```diff
@@ -67,45 +71,55 @@ Example notifications plus help with all available arguments:
                      [--logfile LOGFILE] [--syslog] [--syslog-local [0-7]]
                      [--disable-stderr] [--datadir DATADIR]
                      [--maxpeers MAXPEERS] [--wallet WALLET] [--host HOST]
+                     [--extended-rpc]
 
     optional arguments:
-    -h, --help            show this help message and exit
-    --datadir DATADIR     Absolute path to use for database directories
-    --maxpeers MAXPEERS   Max peers to use for P2P Joining
-    --wallet WALLET       Open wallet. Will allow you to use methods that
-                          require an open wallet
-    --host HOST           Hostname ( for example 127.0.0.1)
+      -h, --help            show this help message and exit
+      --datadir DATADIR     Absolute path to use for database directories
+      --maxpeers MAXPEERS   Max peers to use for P2P Joining
+      --wallet WALLET       Open wallet. Will allow you to use methods that
+                            require an open wallet
+      --host HOST           Hostname ( for example 127.0.0.1)
+      --extended-rpc        Use extended json-rpc api
 
     Network options:
-    --mainnet             Use MainNet
-    --testnet             Use TestNet
-    --privnet             Use PrivNet
-    --coznet              Use CozNet
-    --config CONFIG       Use a specific config file
+      --mainnet             Use MainNet
+      --testnet             Use TestNet
+      --privnet             Use PrivNet
+      --coznet              Use CozNet
+      --config CONFIG       Use a specific config file
 
     Mode(s):
-    --port-rpc PORT_RPC     port to use for the json-rpc api (eg. 10332)
-    --port-rest PORT_REST   port to use for the rest api (eg. 80)
+      --port-rpc PORT_RPC   port to use for the json-rpc api (eg. 10332)
+      --port-rest PORT_REST
+                            port to use for the rest api (eg. 80)
 
     Logging options:
-    --logfile LOGFILE     Logfile
-    --syslog              Log to syslog instead of to log file ('user' is the
-                          default facility)
-    --syslog-local [0-7]  Log to a local syslog facility instead of 'user'.
-                          Value must be between 0 and 7 (e.g. 0 for 'local0').
-    --disable-stderr      Disable stderr logger
+      --logfile LOGFILE     Logfile
+      --syslog              Log to syslog instead of to log file ('user' is the
+                            default facility)
+      --syslog-local [0-7]  Log to a local syslog facility instead of 'user'.
+                            Value must be between 0 and 7 (e.g. 0 for 'local0').
+      --disable-stderr      Disable stderr logger
+
+Running Extended Json-RPC Api Server
+""""""""""""""""""""""""""""""""""""
 
+Using the ``--extended-rpc`` arg will start your server with additional methods from ExtendedJsonRpcApi in addition to the normal JsonRpcApi methods. All original functionality is retained with the added functionality from ExtendedJsonRpcApi.
```

Comment:
As pointed out in my previous comment on the console argument, if you've not read the `ExtendedJsonRpcApi` code then this will be a somewhat vague description. The second line also repeats what was already said in the first, just using different wording. 

I think it would help the reader if we point out that this flag enables additional unofficial RPC methods. Saying that will immediately make it clear
1) that the default methods are officially supported methods
2) that the extra methods are unofficial 
3) that they `additional` , thus retaining the other normal functionality. (somewhat implied already be `extended`)
4) why there is a separation.

Reply:
I removed this section because updating the `help` for the `--extended-rpc` flag eliminates the confusion and the need for extra explanation I think.